tests/regression/test_staking_router_stake_distribution.py

```diff
@@ -209,7 +209,10 @@ def test_target_share_distribution(stranger):
     )
     # add enough depositable keys to the target module to overcome the target share
     # at least first 3 NOs, each with 1/3 of the `keys_to_allocate_double` available keys
-    fill_simple_dvt_ops_vetted_keys(stranger, 3, (module.deposited_keys + keys_to_allocate_double + 3) // 3)
+    if module.id == 2:
```

Comment:
I am fine with that as a quick fix for the next release.
We can rewrite it later once we migrate to the new repo.